pyproject.toml

```diff
@@ -0,0 +1,113 @@
+[build-system]
+build-backend = "setuptools.build_meta:__legacy__"
+requires = [
+    "setuptools>=64.0.0",
+    "tomli  ; python_version < '3.11'",
+]
+
+[project]
+name = "dask-cuda"
+dynamic = [
+    "version",
+    "dependencies",
+]
+description = "Utilities for Dask and CUDA interactions"
+readme = { file = "README.md", content-type = "text/markdown" }
+authors = [
+    { name = "NVIDIA Corporation" },
+]
+license= { text = "Apache-2.0" }
+requires-python = ">=3.8"
+classifiers=[
+    "Intended Audience :: Developers",
+    "Topic :: Database",
+    "Topic :: Scientific/Engineering",
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+]
+
+[project.scripts]
+dask-cuda-worker = "dask_cuda.cli.dask_cuda_worker:go"
+dask-config = "dask_cuda.cli.dask_config:go"
+
+[project.urls]
+Homepage = "https://github.com/rapidsai/dask-cuda"
+
+[tool.coverage.run]
+disable_warnings = "include-ignored"
+include = "dask_cuda/*"
+omit = "dask_cuda/tests/*"
+
+[tool.isort]
+line_length = 88
+multi_line_output = 3
+include_trailing_comma = true
+force_grid_wrap = 0
+combine_as_imports = true
+order_by_type = true
+known_dask = [
+    "dask",
+    "distributed",
+]
+known_rapids = [
+    "rmm",
+    "cuml",
+    "cugraph",
+    "dask_cudf",
+    "cudf",
+    "ucp",
+]
+known_first_party = [
+    "dask_cuda",
+]
+default_section = "THIRDPARTY"
+sections = [
+    "FUTURE",
+    "STDLIB",
+    "THIRDPARTY",
+    "DASK",
+    "RAPIDS",
+    "FIRSTPARTY",
+    "LOCALFOLDER",
+]
+skip = [
+    ".eggs",
+    ".git",
+    ".hg",
+    ".mypy_cache",
+    ".tox",
+    ".venv",
+    "build",
+    "dist",
+    "__init__.py",
```

Comment:
Just like with flake8, do we need to skip `__init__.py`?

Reply:
This is just taken from this line

https://github.com/rapidsai/dask-cuda/blob/2c99f5a0fdf59a7f89e182c4add9121e0b8dd48d/setup.cfg#LL54